packages/core/src/util/renderer.ts

```diff
@@ -172,6 +172,14 @@ export interface WithClassname {
   className?: string;
 }
 
+export interface EnumOption {
+  label: string;
+  description: string;
+  value: any;
+}
+
+export const enumToEnumOptionMapper = (e: any): EnumOption => ({ label: e, description: e, value: e });
```

Comment:
we should `stringify` `e` for label and description as we typed them as `string`